src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -0,0 +1,148 @@
+package org.testeditor.fixture.web
+
+interaction type wait {
+	label = "Wait seconds"
```

Comment:
- new discussion point:
I think a label is really helpful when differentiating -> starting with different browser versions.
like this:

	element Firefox is BrowserApplication {
		label = "Firefox(47 and above)"
		locator ="modernfirefox"
	}
as on other hands without a label
like this
	element Firefox is BrowserApplication {
		locator ="modernfirefox"
	}
	element OlderFirefox is BrowserApplication {
		locator ="firefox"
	}

what does OlderFirefox without a label means (every developer, even when he was not involved into these discussions, has to know about the past of the development status. But with a label, he just know, that with this element, just Firefox versions older then  V. 47 are on focus.

I think at this point a label is very helpful.
What do you think ?